src/components/AppNavigationCaption/AppNavigationCaption.vue

```diff
@@ -1,35 +1,82 @@
+<docs>
+```
+<AppNavigationCaption
+	title="Your caption goes here">
+	<ActionButton
+		slot="actions"
+		icon="icon-add"/>
+</AppNavigationCaption>
+```
+</docs>
+
 <template>
 	<li class="app-navigation-caption">
-		{{ title }}
+		<!-- Title of the caption -->
+		<div class="app-navigation-caption__title">
+			{{ title }}
+		</div>
+
+		<!-- Actions -->
+		<div v-if="hasActions"
+			class="app-navigation-caption__actions">
+			<Actions
+				v-bind="$attrs">
```

Comment:
What is this new way of yours to put single-attr into a new line? While the others are on the same line btw?

Reply:
:P